index.js

```diff
@@ -4,32 +4,236 @@ const logo = document.querySelector('#logo');
-    logo.classList.remove('mobile-back');
-    main.classList.remove('mobile-back');
+    logo.classList.remove('all');
+    main.classList.remove('all');
     union.classList.remove('img_remove');
     cross2.remove();
   });
   navbar.addEventListener('click', () => {
     navbar.classList.remove('mobile-menu');
-    logo.classList.remove('mobile-back');
-    main.classList.remove('mobile-back');
+    logo.classList.remove('all');
+    main.classList.remove('all');
     union.classList.remove('img_remove');
     cross2.remove();
   });
 });
+
+//  pop up window day-2
+
+// function blur() {
+//   const header = document.querySelector(".header");
+//   const mainSection = document.querySelector(".main-section");
+//   const works = document.querySelector(".works");
+//   const about = document.querySelector("#about");
+//   const contact = document.querySelector("#contact");
+
+// header.classList.toggle("all");
+// mainSection.classList.toggle("all");
+// works.classList.toggle("all");
+// about.classList.toggle("all");
+// contact.classList.toggle("all");
+// }
+
+const popup = document.querySelectorAll('.big-btn');
+
+const doc = document.createElement('div');
+doc.setAttribute('id', 'popup_window');
+
+const header = document.querySelector('.header');
+const mainSection = document.querySelector('.main-section');
+const works = document.querySelector('.works');
+const about = document.querySelector('#about');
+const contact = document.querySelector('#contact');
+
+for (let i = 0; i < popup.length; i += 1) {
+  popup[i].addEventListener('click', () => {
+    const cardid = document.querySelectorAll('.cards')[i].id;
+
+    header.classList.add('all');
+    mainSection.classList.add('all');
+    works.classList.add('all');
+    about.classList.add('all');
+    contact.classList.add('all');
+
+    const projects = [
+      {
+        id: 1,
+        key: 'one1',
+        heading: 'Tonic',
+
+        frame: { li1: 'CANOPY', li2: 'Back End Dev', li3: '2019' },
+
+        img: 'images/Snapshoot.png',
+        alt: 'Tonic project',
+        para1: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+        para2: 'Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.',
+
+        language: {
+          li1: 'HTML',
+          li2: 'Css',
+          li3: 'JavaScript',
+          li4: 'bootstrap',
+          li5: 'query',
+          li6: 'react.js',
+        },
```

Comment:
- [ ] Kindly consider adding the `link to live version` and the `link to source` in all the objects in this array.
project requirement:
-  For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.